src/main/java/wolox/training/models/Book.java

```diff
@@ -26,25 +56,52 @@
 
     @Column(nullable = false)
     private String author;
+    public void setAuthor(String author){
```

Comment:
I think this setter could be private, because it's not used from another class (This way we could make this object even more immutable 😃 )